.buildkite/scripts/compile.sh

```diff
@@ -2,4 +2,4 @@
 set -e
 
 yarn install --frozen-lockfile
-yarn tsc
+lerna run build
```

Comment:
I've changed this because each project may have different requirements to be built. As an example, `design-tokens` doesn't need typescript to compile its assets.